### PR TITLE
fix(FTL-6114): updated documentation for did-auth-lib

### PR DESCRIPTION
**Pull request by oleksiipiliugin**
- Undepricate DidAuthService lib class in order to proceed with its usage;
- Introduce up-to-date documentation with code snippets for did-auth-lib.

### CODE REVIEW

File: common-libs/did-auth-lib/README.md [hunk 1]
```diff
@@ -1,36 +1,193 @@
 # Affinidi Did Auth Helpers (PUBLIC)
```
Comment (Volland) [inline common-libs/did-auth-lib/README.md:1]: @oleksiipiliugin why we mention in a headers that helpers is public ?

File: common-libs/did-auth-lib/README.md [hunk 2]
```diff
@@ -1,36 +1,193 @@
...
 
 ## Usage:
+
+This library allows proving that the client/user is the owner of provided `DID`. Based on this proof App builders can give the client/user access to the appropriate resources.
+
+`Did-Auth` flow is similar to the `Sharing VC` flow, but without `VC` exchanges, since `DID` itself should be enough info from the client/user to authenticate.
```
Comment (Volland) [inline common-libs/did-auth-lib/README.md:7]: could we add a flow diagram here ?

File: common-libs/did-auth-lib/README.md [hunk 3]
```diff
@@ -1,36 +1,193 @@
...
-import DidAuthService from '@affinidi/affinidi-did-auth-lib'
 
-const options = { encryptedSeed: '...', encryptionKey: '...' }
-const didAuthService = new DidAuthService(options)
+#### Building of the response token(client side)
 
-// At the Server side (should recieve client did and generate for it didAuthRequest)
-const didAuthRequest = await didAuthService.createDidAuthRequestToken(audienceDid, expiresAt?)
+To create a response token it is recommended to initialize the `AffinidiDidAuthService` class with `encryptedSeed` and `encryptionKey` options on the client side in the same way as for the service side.
+Use `createDidAuthResponseToken` method to create `Did-Auth` response token. It is necessary to have a fresh request token from the service.
```
Comment (Volland) [inline common-libs/did-auth-lib/README.md:72]: do we have any information about exp time ?

File: common-libs/did-auth-lib/README.md [hunk 4]
```diff
@@ -1,36 +1,193 @@
...
+ * environment {Stirng} (required) - environment, could be `dev`, `staging` or `prod`
+ * accessApiKey {Stirng} (required) - client access api key(uuid) 
+ */
+const isValid = await affinidiDidAuthService.verifyDidAuthResponseToken(responseToken, {
+  environment: environment,
+  accessApiKey: accessApiKey,
+})
+```
+
+[Generate accessApiKey](https://apikey.affinidi.com/).
```
Comment (Volland) [inline common-libs/did-auth-lib/README.md:114]: @oleksiipiliugin it is production only . for dev and staging it should be a different links

File: common-libs/did-auth-lib/README.md (hunk 1 above)
Comment (oleksiipiliugin) [reply common-libs/did-auth-lib/README.md:1]: I think it comes here from GitLab where we had such flags to mark if the library is public or private.

File: common-libs/did-auth-lib/README.md (hunk 2 above)
Comment (oleksiipiliugin) [reply common-libs/did-auth-lib/README.md:7]: it is in place pls [see](https://github.com/affinityproject/affinidi-core-sdk/pull/230/files#diff-77696b96a01de859876a036be6039b21a1fb0faeabd96d660f6aa968ab1694e8R17)

**Review by Volland — APPROVED** [review]
LGTM

**Review by anton-iskryzhytskyi — APPROVED** [review]

**Review by cosminm-affnd — APPROVED** [review]

**Pull request merged**